src/targets/registry.ts

```diff
@@ -265,18 +312,14 @@ export class RegistryTarget extends BaseTarget {
     this.logger.info(
       'Adding extra data (checksums, download links) for available artifacts...'
     );
-    const files: { [key: string]: any } = {};
 
+    const files: { [key: string]: any } = {};
     await mapLimit(artifacts, MAX_DOWNLOAD_CONCURRENCY, async artifact => {
       const fileData = await this.getArtifactData(artifact, version, revision);
-      if (Object.keys(fileData).length > 0) {
```

Comment:
This protection is no longer necessary because the new condition above (`this.registryConfig.checksums.length === 0`) is equivalent, yes?

Reply:
Yup 👍🏻